docs/subscriptions/update.md

```diff
@@ -0,0 +1,162 @@
+# Update a subscription
+
+```text
+PATCH /subscriptions/{guid}
+```
+
+This endpoint allows clients to update information about a subscription. It returns the following information:
```

Comment:
Might it make it easier to read/digest if first the patch calls are explained, and only after what the end point returns? (I feel it'd also make it easier to find the relevant info when looking for how to patch a subscription.)